www-servers/pound/pound-3.0.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+MY_P=${P/p/P}
+DESCRIPTION="A http/https reverse-proxy and load-balancer"
+HOMEPAGE="https://www.apsis.ch/pound.html"
+SRC_URI="http://www.apsis.ch/pound/${MY_P}.tgz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~mips ~x86"
+
+DEPEND="
+	dev-libs/libpcre
+	dev-libs/libyaml
+	dev-libs/nanomsg
+	dev-libs/openssl
+	net-libs/mbedtls
+"
+
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/${MY_P}"
+
+src_compile() {
+	rm -r GPL.txt || die
+	cmake_src_compile
+}
+
+src_install() {
+	dodir /usr/sbin
+	cp "${BUILD_DIR}"/pound "${ED}"/usr/sbin/ || die
+	doman "${S}"/man/pound.8
+	dodoc README.md
+
+	dodir /etc/init.d
+	newinitd "${FILESDIR}"/pound.init-1.9 pound
+
+	insinto /etc
+	newins "${FILESDIR}"/pound-2.2.cfg pound.cfg
+}
```

Comment:
What would `cmake_src_install` do here? I'd imagine you could replace many of these steps with that single call.

Reply:
It does not work because of -> ninja: error: unknown target 'install'

Reply:
I wonder if it works with `CMAKE_MAKEFILE_GENERATOR="emake"`

Reply:
(note that I haven't actually checked the upstream build system at all, which would tell this instantly)

Reply:
The CMakeLists.txt does not offer any install way, only compilation

Reply:
I leave it as is due to that